src/containers/Auth/LoginForm/index.js

```diff
@@ -18,42 +18,63 @@ import {reduxForm} from 'redux-form';
 })
 
 export default class LoginForm extends Component {
+  static propTypes = {
+    fields: PropTypes.object.isRequired,
+    error: PropTypes.object,
```

Comment:
Getting a warning.js:45Warning: Failed propType: Invalid prop `error` of type `string` supplied to `LoginForm`, expected `object`. Check the render method of `ReduxForm(LoginForm)` here.